package.json

```diff
@@ -122,6 +122,7 @@
     "check:socks5": "mocha --config test/manual/mocharc.json test/manual/socks5.test.ts",
     "check:csfle": "mocha --config test/mocha_mongodb.json test/integration/client-side-encryption",
     "check:snappy": "mocha test/unit/assorted/snappy.test.js",
+    "eslint:fix": "npm run check:eslint -- --fix",
```

Comment:
I don't mind the addition but any chance we could use the check prefix? `check:fix`,`check:fixlint`, `check:fixeslint`

Reply:
what about `fix:eslint`?  None of our other `check` prefixes actually modify any code, so it doesn't feel right to use `check` as the prefix.

Reply:
Sure! that seems good to me I could see other fixes being added, like maybe a fix to run prettier on all our readme(s)